includes/admin/class-sensei-onboarding.php

```diff
@@ -19,6 +19,8 @@
  * @since   3.1.0
  */
 class Sensei_Onboarding {
+	const SKIP_SETUP_WIZARD_OPTION      = 'sensei_skip_setup_wizard';
```

Comment:
Do we need this option? Seems like it would be enough to check if the wizard was completed or not, and show the notice if not. In this case, "completed" means that they made it to the _Ready_ step, regardless of whether or not they took any action there.

Reply:
I liked the @yscik idea. https://github.com/Automattic/sensei/pull/3096#issuecomment-622506150

Do you think it's good?

Reply:
It doesn't matter much to me how it's implemented, as long as it meets the requirements. 🙂